packages/translate/src/index.js

```diff
@@ -166,10 +166,12 @@ Translate.prototype.detect = function(input, callback) {
  *
  * @resource [Discover Supported Languages]{@link https://cloud.google.com/translate/v2/using_rest#supported-languages}
  *
+ * @param {options=} options - Configuration options.
+ * @param {options} options.target - Configuration options.
  * @param {function} callback - The callback function.
  * @param {?error} callback.err - An error returned while making this request.
- * @param {string[]} callback.languages - The supported ISO 639-1 language
- *     codes.
+ * @param {string[]|object[]} callback.languages - The supported ISO 639-1
```

Comment:
Let's just always return an array of strings. If the user wants the display name property, they can get that from `apiResponse`.